app/org/sagebionetworks/bridge/dynamodb/DynamoStudy.java

```diff
@@ -48,11 +48,13 @@
     private EmailTemplate resetPasswordTemplate;
     private boolean strictUploadValidationEnabled;
     private boolean healthCodeExportEnabled;
+    private Long minSupportedVersion;
 
     public DynamoStudy() {
         profileAttributes = new HashSet<>();
         taskIdentifiers = new HashSet<>();
         dataGroups = new HashSet<>();
+        minSupportedVersion = 0L;
```

Comment:
I think you want this null by default. If it doesn't exist, we won't enforce a minimum.